plot_continuous_monitor.py

```diff
@@ -81,15 +81,13 @@ def plot_continuous_monitor(filename, open_graphs=False):
     xvdf_running_disk_monotasks = 0
     xvdb_running_disk_monotasks = 0
     if "Running Disk Monotasks" in json_data:
-      # Parse the number of currently running disk monotasks for each disk. xvdf is mounted on /mnt2
-      # and xvdb on /mnt (the indexing below will need to be updated if we use different instance
-      # types).
+      # Parse the number of currently running disk monotasks for each disk.
```

Comment:
What about doing something like:
(first rename running_disk_monotasks_info to be plural)

for running_monotasks_info in running_disk_monotasks_infos:
  disk_name = running_monotasks_info["Disk Name"]
  running _monotasks = running_monotasks_info["..."]
  if disk_name == "xvdb":
    xvdb_running_disk_monotasks = running_monotasks
  elif disk_name == "xvdf":
    xvdf_running....

This is a bit more verbose but I think easier to read